lib/easypost/version_info.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Retrieve operating system information
+module EasyPost::VersionInfo
```

Comment:
Issue: This should be named something like `OsVersionInfo`, otherwise we won't know if it's language, client, OS, other, etc